packages/manager/src/features/CloudPulse/Dashboard/CloudPulseDashboardLanding.tsx

```diff
@@ -0,0 +1,37 @@
+import { Grid, Paper } from '@mui/material';
+import * as React from 'react';
+
+import { CircleProgress } from 'src/components/CircleProgress';
+
+import { GlobalFilters } from '../Overview/GlobalFilters';
+import { useLoadUserPreferences } from '../Utils/UserPreference';
+import { CloudPulseDashboard } from './CloudPulseDashboard';
+
+import type { FiltersObject } from '../Overview/GlobalFilters';
+import type { TimeDuration } from '@linode/api-v4';
+
+export const CloudPulseDashboardLanding = () => {
+  // eslint-disable-next-line @typescript-eslint/no-empty-function
+  const onFilterChange = React.useCallback((_filters: FiltersObject) => {}, []);
+  const { isLoading } = useLoadUserPreferences();
```

Comment:
Why do we have a wrapper function when we can just use the query directly?

```suggestion
  const { isLoading } = usePreferences();
```

Reply:
done, thanks

Reply:
We are using a separate utils file to maintain the preferences retrieve & update logics. There we are also maintaining a copy of fetched data that we are using in different components to fetch preferences instead of calling query in every component.